cov_mult: 10.0
downSampleSize: 200
feature_list: [unimodal_ftForce, crossmodal_targetEEDist]
rf_center: kinEEPos
rf_radius: 0.15
sgd_w_positive: 15.0
states: 10
svm_w_positive: 20.0
